packages/p2p-media-loader-core/src/utils/peer.ts

```diff
@@ -1,5 +1,6 @@
 import md5 from "nano-md5";
 import { utf8ToUintArray } from "./utils";
+import packageJson from "../../../../package.json";
```

Comment:
I did a small research. Reading `package.json` directly is not a good idea because it includes the whole file in the bundle. It is better to read individual fields in build scripts and embed them like described here: https://stackoverflow.com/questions/67194082/how-can-i-display-the-current-app-version-from-package-json-to-the-user-using-vi